composer.json

```diff
@@ -35,22 +35,21 @@
         "guzzlehttp/guzzle": "^7.2",
         "mockery/mockery": "^1.3.2",
         "monolog/monolog": "*",
-        "sensio/framework-extra-bundle": "^4.0 || ^5.5.1 || ^6.0",
         "doctrine/annotations": "^1.11",
-        "symfony/browser-kit": "^4.4 || ^5.0 || ^6.0",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/finder": "^4.4 || ^5.0 || ^6.0",
-        "phpunit/phpunit": "^9.6.15",
```

Comment:
please don't revert this change. i ditched phpunit-bridge in favor of directly using phpunit.

Reply:
i will check. i just accept my modification when i rebase the branch.